client/my-sites/woocommerce/landing-page.tsx

```diff
@@ -153,12 +152,16 @@ const LandingPage: React.FunctionComponent< Props > = ( { siteId } ) => {
 		};
 	}
 
+	const { isAboveElement, targetRef: ctaRef, referenceRef: headerRef } = useScrollAboveElement();
+
 	return (
 		<div className="landing-page">
-			<FixedNavigationHeader navigationItems={ navigationItems } contentRef={ ctaRef }>
-				<Button onClick={ onCTAClickHandler } primary disabled={ isTransferringBlocked }>
-					{ displayData.action }
-				</Button>
+			<FixedNavigationHeader navigationItems={ navigationItems } ref={ headerRef }>
+				{ isAboveElement && (
```

Comment:
@Rolilink we might want to reverse the naming on `isAboveElement` and make it `isBelowElement` which is how it should read in this usage: "if target is below reference, show the target". Alternatively we could rename it to `isReferenceAboveTarget` so there isn't any room for confustion.

Reply:
@lsl, I like the idea, feel free to do this one but it will probably affect the marketplace search so make sure to make the change there too.